### PR TITLE
REF: Use fs module for screens that show action sheet

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2xexqkdb19tnw0pcu542ee2ke4

[download apk](http://filestorage.bluewallet.io:1488/1607905696.apk)

File: screen/send/psbtMultisigQRCode.js [hunk 1]
```diff
@@ -52,9 +52,11 @@ const PsbtMultisigQRCode = () => {
 
     ActionSheet.showActionSheetWithOptions({ options, cancelButtonIndex: 0 }, async buttonIndex => {
       if (buttonIndex === 1) {
-        fs.takePhotoWithImagePickerAndReadPhoto.then(onBarScanned);
```
Comment (Overtorment) [inline screen/send/psbtMultisigQRCode.js:55]: whats the difference between 
`then(onBarScanned);` and `then(data => onBarScanned({ data }));`
?
just to wrap in `{}`  ?

File: screen/wallets/viewEditMultisigCosigners.js [hunk 2]
```diff
@@ -402,30 +400,32 @@ const ViewEditMultisigCosigners = () => {
     });
   };
 
+  const showActionSheet = () => {
```
Comment (Overtorment) [inline screen/wallets/viewEditMultisigCosigners.js:403]: still feels like this could be implemented only once somewhere

**Review by Overtorment — APPROVED** [review]

File: screen/send/psbtMultisigQRCode.js (hunk 1 above)
Comment (marcosrdz) [reply screen/send/psbtMultisigQRCode.js:55]: Was failing without it

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/uu0jzhej3gn8zt368bdrune7q8

[download apk](http://filestorage.bluewallet.io:1488/1607963444.apk)

File: screen/wallets/viewEditMultisigCosigners.js (hunk 2 above)
Comment (marcosrdz) [reply screen/wallets/viewEditMultisigCosigners.js]: done

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rbtge5pr6wjfau23y3xr5ggg3w

[download apk](http://filestorage.bluewallet.io:1488/1608006436.apk)

**Pull request merged**